db/seeds.rb

```diff
@@ -1,7 +1,22 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+require 'faker'
+
+20.times do |i|
+	Venue.create({address: Faker::Address.full_address,
+                description: Faker::Lorem.sentence,
+                people_capacity: rand(1...99)})
+end
+
+puts 'Created 20 venues'
```

Comment:
too optimistic

Reply:
i removed it, but as for me it's good to control in console what have been created before by seeds.